pkg/store/mastership/election.go

```diff
@@ -168,13 +169,13 @@ func (e *atomixDeviceMastershipElection) watchElection(ch <-chan *election.Event
 	}
 }
 
-func (e *atomixDeviceMastershipElection) isMaster() (bool, error) {
+func (e *atomixDeviceMastershipElection) getMastership() *Mastership {
 	e.mu.RLock()
 	defer e.mu.RUnlock()
 	if e.mastership == nil || string(e.mastership.Master) != e.election.ID() {
```

Comment:
I assume we still need to keep this condition

Reply:
This condition should not be here. We're returning the mastership regardless of whether the local node is the master, but this check only returns the mastership if the local node is the master. Other nodes should have access to mastership info even if they're not the master.

Reply:
Makes sense. So it should be moved to where it needs to be checked based on another comment.

Reply:
Yeah... the local node ID can be read from the mastership store interface to check it.